### PR TITLE
Run tests with coverage

### PR DESCRIPTION
**Pull request by hugovk**
This uses the Coverage.py tool to measure the coverage of the unit tests.

Coverage is currently an impressive 94%!
https://coveralls.io/builds/8570591

It gives a report of the coverage on the CI with `coverage report`:
https://travis-ci.org/hugovk/Gutenberg/jobs/171618837#L211

And also sends the data to Coveralls, which is handy to track progress and show coverage of individual files:
https://coveralls.io/github/hugovk/Gutenberg

Please could an admin also enable Coveralls at https://coveralls.io/repos/new? It's free for open source :)

If you're developing locally, you can use `coverage html` to generate similar reports in htmlcov/index.html

### CODE REVIEW

**Comment by hugovk** [comment]
The build failed due to unsupported Python 3.2, see https://github.com/c-w/Gutenberg/issues/49. The others passed.

File: .travis.yml [hunk 1]
```diff
@@ -27,4 +27,13 @@ install:
   - pip install -r requirements.pip
   - pip install -r requirements-py3.pip
 
-script: nosetests
+  # Coverage 4.0 doesn't support Python 3.2
+  - if [ "$TRAVIS_PYTHON_VERSION" == "3.2" ]; then travis_retry pip install coverage==3.7.1; fi
```
Comment (sethwoodworth) [inline .travis.yml]: Please remove now that your PR to remove python3.2 has been merged.

**Comment by hugovk** [comment]
@sethwoodworth Merge conflict resolved, build passes.

**Comment by sethwoodworth** [comment]
Tested locally, LGTM!  Merging.

**Pull request merged**